portality/scripts/publisher_email_in_doaj.py

```diff
@@ -0,0 +1,94 @@
+""" Creates a CSV of publisher email addresses """
+
+import os, codecs
+import esprit
+from portality.core import app
+from portality.models import Account, Journal
+from portality.clcsv import UnicodeWriter
+from portality.lib import dates
+
+source = {
+    "host": app.config.get("ELASTIC_SEARCH_HOST"),
+    "index": app.config.get("ELASTIC_SEARCH_DB")
+}
+
+conn = esprit.raw.Connection(source.get("host"), source.get("index"))
+
+publisher_query = {
+    'query': {
+        'term': {
+            'role.exact': 'publisher'
+        }
+    }
+}
+
+
+def publishers_with_journals():
+    """ Get accounts for all publishers with journals in the DOAJ """
+    for acc in esprit.tasks.scroll(conn, 'account', q=publisher_query):
+        account = Account(**acc)
+        journal_ids = account.journal
+        if journal_ids is not None:
```

Comment:
Surprised to see the underlying method return a list of values when there are journals and `None` rather than `[]` when there are none to return. Don't have to fix it in this PR.